### DISCUSSION TITLE
Get items fom the FPS Fest 2024

### DISCUSSION THREAD

**Discussion by woctezuma** [chr233/ASFEnhance#249] [杂谈 / General]
- https://store.steampowered.com/category/action_fps
- https://store.steampowered.com/points/shop/event/sale_fps_fest
- https://store.steampowered.com/points/shop/app/2855100
- https://www.steamcardexchange.net/index.php?gamepage-appid-2855100
- https://steamdb.info/app/2855100/communityitems/

Command:
```sh
!rpi ASF 280938,280939,280940
```

**Comment by kudrman1978** [comment]
defIds is incorrect!

**Reply by chr233** [reply]
didn't you see "todo"?

**Reply by woctezuma** [reply]
The event will start on April 15, 2024. Until then, the ids are unknown.

**Comment by aleatorio1337** [comment]
I tried this command but it didn't work, has anyone discovered the codes?

**Reply by woctezuma** [reply]
Of course.

**Reply by botka4aet** [reply]
Wait id for items here - https://steamdb.info/app/2855100/info/#assets
